vitrina/templates/component/navbar.html

```diff
@@ -73,11 +73,22 @@
                     {% endif %}
                 </div>
                 <div class="navbar-item">
-                    {% if locale == "en" %}
-                        <a href="?lang=lt" class="button is-text" title="{% translate "lang.lt" %}">LT</a>
-                    {% else %}
-                        <a href="?lang=en" class="button is-text" title="{% translate "lang.en" %}">EN</a>
-                    {% endif %}
+                    <div class="navbar-item has-dropdown is-hoverable">
+                        <a class="navbar-link"><strong>{{ request.session.LANGUAGE_CODE }}</strong></a>
```

Comment:
```suggestion
                        {% get_current_language as LANGUAGE_CODE %}
                        <a class="navbar-link"><strong>{{ LANGUAGE_CODE }}</strong></a>
```

Use `get_current_language` template tag instead.

https://docs.djangoproject.com/en/3.2/topics/i18n/translation/#switching-language-in-templates